game/scripts/vscripts/abilities/twin/boss_twin_twin_empathy.lua

```diff
@@ -0,0 +1,7 @@
+LinkLuaModifier("modifier_boss_twin_twin_empathy_buff", "abilities/twin/modifier_boss_twin_twin_empathy.lua", LUA_MODIFIER_MOTION_NONE)
```

Comment:
You can remove this file and just make the ability data driven, it doesn't do anything and is only there for KV data and tooltips.